app/src/main/java/com/babylon/wallet/android/utils/BiometricExtensions.kt

```diff
@@ -84,6 +84,10 @@ suspend fun FragmentActivity.biometricAuthenticateSuspend(): Boolean {
     }
 }
 
+enum class BiometricAuthenticationResult {
+    Succeeded, Error, Failed
```

Comment:
What is the difference between error and failed?

Maybe `Granted` and `Denied` suit better for result names?

Reply:
failed -> wrong fingerprint
error -> user dismiss auth dialog (or any other errors)

>Maybe Granted and Denied suit better for result names?
I kept the same names as the native callbacks
onAuthenticationSucceeded, onAuthenticationError, onAuthenticationFailed

but I like also the granted/denied. I am ok if you want to rename them.

Reply:
Ah that's ok thanks for the explanation.